### ISSUE TITLE
"Latest" label on very old release

### ISSUE THREAD

**Issue by michacassola** [apache/incubator-pagespeed-ngx#1730]
Hi, please move the latest label to the actual latest stable release.

**Comment by dvershinin** [comment]
@michacassola if you just want to get the latest stable version source, you can use [lastversion](https://lastversion.getpagespeed.com/):

```bash
lastversion apache/incubator-pagespeed-ngx --source
#> https://github.com/apache/incubator-pagespeed-ngx/archive/v1.13.35.2-stable/incubator-pagespeed-ngx-v1.13.35.2-stable.tar.gz
```

Or even:

```bash
lastversion download apache/incubator-pagespeed-ngx 
```

**Comment by oschaaf** [comment]
Well.. I think we could do better and re-tag to 1.14; however up to now there hasn't been a proper location to host the psol binary we need to link against. I think we do have that these days when we release mod_pagespeed 1.14, as a prebuilt version has been dropped in the artifacts directory as a courtesy.
So we could probably make some changes in this repo to get 1.14 to work. A good example resides over at https://github.com/sambacha/nginx-autoinstall/pull/1/files.

**Comment by oschaaf** [comment]
Yeah here it is: https://dist.apache.org/repos/dist/release/incubator/pagespeed/1.14.36.1/x64/
I should note that there's no 32 bit variant, as I ran into a bit of a hassle to get that to through the build flow.

**Comment by dvershinin** [comment]
Isn't all that is needed now is simply [tagging another release](https://github.com/apache/incubator-pagespeed-ngx/releases) "v1.14.33.1" (without "RC1" in it)?

**Comment by oschaaf** [comment]
> Isn't all that is needed now is simply [tagging another release](https://github.com/apache/incubator-pagespeed-ngx/releases) "v1.14.**36**.1"?

Checking [this commit](https://github.com/apache/incubator-pagespeed-ngx/commit/4aeb203468d55e495ac9242e2c01a90da52ce5d6#diff-2b4a325bf93d5c063cf176fb8666053e3f304ff118a0d22a8511d907d4a6009e) it looks like I did modify the psol download location for branch 36, so indeed that part is settled. 

However to truly release this under the ASF incubator umbrella, this 
- Needs to a VOTE thread on the ASF mailing lists to give folks an opportunity to review how things are with respect to ASF policies. I tried to get things in order here, but this repo didn't get as much love as mod_pagespeed; still life is also much simpler in this repo, so it may just pass.
- When the vote passes, the rest of the ASF release procedure needs to be followed. I do think ASF recently started allowing git-based releases which could make things a little simpler here.

**Comment by JeanVEGA** [comment]
> I tried to get things in order here, but this repo didn't get as much love as mod_pagespeed

@oschaaf, is there anything specific that needs to be sorted out? How is the vote thread created?
